css/icons.scss

```diff
@@ -9,7 +9,7 @@
 @include icon-black-white('lobby', 'spreed', 1);
 @include icon-black-white('text', 'filetypes', 1, true);
 
-.app-Talk,
+.app-talk,
```

Comment:
ahahahha!
That's why you never use capital letters in class/id

Reply:
Well, then enforce this on your property?
https://github.com/nextcloud/spreed/commit/72edd115cccf18494ce54d08cf3cc801141586d7
*cough*

It works well. But of course if you just change the template and not the css it doesn't match anymore?